yt_dlp/extractor/odnoklassniki.py

```diff
@@ -226,6 +235,14 @@ class OdnoklassnikiIE(InfoExtractor):
         'skip': 'Site no longer embeds',
     }]
 
+    def _clear_cookies(self, cdn_url):
+        # Direct http downloads will fail if CDN cookies are set
+        # so we need to reset them after each format extraction
+        if self._get_cookies('https://notarealsubdomain.mycdn.me/'):
+            self.cookiejar.clear(domain='.mycdn.me')
+        if self._get_cookies(cdn_url):
+            self.cookiejar.clear(domain=urllib.parse.urlparse(cdn_url).netloc)
+
```

Comment:
I don't think the if should be needed here. But I'll investigate it separately.

Reply:
If there are no cookies scoped to the domain argument, `clear` will keyerror

Reply:
I know. Saying we should improve that.